test/integration/targets/flatpak_remote/aliases

```diff
@@ -1,4 +1,4 @@
-unsupported
+shippable/posix/group3
```

Comment:
The `needs/privileged` alias will need to be removed for this to run on Shippable.

I don't know why the original test included it, but hopefully it's not needed.

Reply:
Well, since docker does not support container nesting, the container needs to be privileged so that flatpak can control all the sandboxing features. When I tried running this in a non-privileged container before, I did get warning messages in flatpak. But it might be that they are irrelevant to the function of these tests. I will test this out again.

Reply:
If it does end up needing to be privileged, we may still be able to run this on Shippable. We currently do not run tests that need to be privileged because in the past we've have issues with stability and performance as a result. However, it's been quite a while since we tested that and our docker hosts have been significantly updated since then.

Reply:
My tests have shown no problems with using a non-privileged containers for the `flatpak_remote`-tests, so I removed the `privileged`-requirement.

When I'm done with the `flatpak_remote` tests, I will rework the `flatpak` tests in the same way and it might be a different situation there. But we will see about that then.